app/main.py

```diff
@@ -16,31 +18,31 @@ def hello():
 
 @app.get("/knowledge_panel")
 def knowledge_panel(
-    facet_name: FacetName,
-    facet_value: Union[str, None] = None,
+    facet_tag: FacetName,
+    value_tag: Union[str, None] = None,
+    lang_code: Union[str, None] = None,
     country: Union[str, None] = None,
 ):
     """
     FacetName is the model that have list of values
     facet_value are the list of values connecting to FacetName eg:- category/beer, here beer is the value
     """
+    lang = TranslationStore.lang(lang_code)
```

Comment:
@alexgarel I want to use this `lang` value in my [i18n.py](https://github.com/openfoodfacts/facets-knowledge-panels/pull/18/files#diff-3a40654f02707d2912abff8e55fc35240fdabfec2dac01d3881e597d394e648f) as language, can you help me out with this?

Reply:
see https://github.com/openfoodfacts/facets-knowledge-panels/pull/18/files#r951326228